sdk/common/src/main/java/io/opentelemetry/sdk/common/export/ConfigBuilder.java

```diff
@@ -158,4 +159,91 @@ protected static Double getDoubleProperty(String name, Map<String, String> map)
       return null;
     }
   }
+
+  /**
+   * Get a duration property from the map, {@code null} if it cannot be found or it has a wrong
+   * type.
+   *
+   * <p>Durations can be of the form "{number}{unit}", where unit is one of:
+   *
+   * <ul>
+   *   <li>ms
+   *   <li>s
+   *   <li>m
+   *   <li>h
+   *   <li>d
+   * </ul>
+   *
+   * @param name The property name
+   * @param map The map where to look for the property
+   * @return the {@link Long} value of the property in milliseconds, {@code null} in case of error
+   *     or if the property cannot be found.
+   */
+  @Nullable
+  protected static Long getDurationMillisProperty(String name, Map<String, String> map) {
```

Comment:
I guess we could just return a `Duration`

Reply:
Done.  Not sure if it's too heavyweight, but it is convenient.